src/azure-openapi-validator/rules/DescriptionMustNotBeNodeName.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { rules } from "../rule";
+
+rules.push({
```

Comment:
Yes there is , though extending isn't exactly the same as how it works in C#, we can only specify range of values for the properties.